Makefile

```diff
@@ -319,7 +319,16 @@ build-minimal:
 build-api:
 	BUILD_GRPC_FOR_BACKEND_LLAMA=true BUILD_API_ONLY=true GO_TAGS=none $(MAKE) build
 
-dist: build
+dist:
+	STATIC=true $(MAKE) backend-assets/grpc/llama-cpp-avx2
```

Comment:
@mudler are you sure `STATIC` works? i am not seeing the LDFLAGS are set during the build

Reply:
also in here, i think this is just the backend build so it doesn't call go anyway?

Reply:
have to check that and get back at you - IIRC was used behind the scenes when compiling llama-cpp, but might be completely useless now

Reply:
ouch, you are right =) nevermind me! this is not needed indeed and can be removed